### DISCUSSION TITLE
Node.JS keeps updating to V17 and killing Wiki.JS

### DISCUSSION THREAD

**Discussion by Network228** [requarks/wiki#5299] [Help / Questions]
Using...

Wiki.js 2.5.255
Node.js 16.15.0
Windows Server 2016

Have been using Wiki.js without issue for over a year and all of a sudden Node.js will randomly, (about every 3 days), update itself to Node.js V17 killing Wiki. :(

Node.js V18 doesn't work with Wiki.js V2 so I cannot just jump over V17 to avoid the issue.

Is this being caused by NPM and how do I get it to stop updating Node.js?

**Comment by NGPixel** [comment]
You must have an auto-update feature enabled on Node.js or some other updater. This isn't caused by Wiki.js.
